airflow/CHANGELOG.md

```diff
@@ -2,94 +2,175 @@
 
 ## 3.2.0 / 2022-09-16 / Agent 7.40.0
 
-* [Added] Update HTTP config spec templates. See [#12890](https://github.com/DataDog/integrations-core/pull/12890).
+***Added***: 
+
+* Update HTTP config spec templates. See [#12890](https://github.com/DataDog/integrations-core/pull/12890).
+
 
 ## 3.1.1 / 2022-06-27 / Agent 7.38.0
 
-* [Fixed] Update mapping and tags in README.md. See [#11967](https://github.com/DataDog/integrations-core/pull/11967). Thanks [moshederri](https://github.com/moshederri).
-* [Fixed] Update airflow integration dogstatsd_mapper_profiles example. See [#12101](https://github.com/DataDog/integrations-core/pull/12101). Thanks [jessezhang91](https://github.com/jessezhang91).
+***Fixed***: 
+
+* Update mapping and tags in README.md. See [#11967](https://github.com/DataDog/integrations-core/pull/11967). Thanks [moshederri](https://github.com/moshederri).
+* Update airflow integration dogstatsd_mapper_profiles example. See [#12101](https://github.com/DataDog/integrations-core/pull/12101). Thanks [jessezhang91](https://github.com/jessezhang91).
+
 
 ## 3.1.0 / 2022-04-05 / Agent 7.36.0
 
-* [Added] Add metric_patterns options to filter all metric submission by a list of regexes. See [#11695](https://github.com/DataDog/integrations-core/pull/11695).
-* [Fixed] Remove outdated warning in the description for the `tls_ignore_warning` option. See [#11591](https://github.com/DataDog/integrations-core/pull/11591).
+***Added***: 
+
+* Add metric_patterns options to filter all metric submission by a list of regexes. See [#11695](https://github.com/DataDog/integrations-core/pull/11695).
+
+***Fixed***: 
+
+* Remove outdated warning in the description for the `tls_ignore_warning` option. See [#11591](https://github.com/DataDog/integrations-core/pull/11591).
+
 
 ## 3.0.0 / 2022-02-19 / Agent 7.35.0
 
-* [Added] Add `pyproject.toml` file. See [#11312](https://github.com/DataDog/integrations-core/pull/11312).
-* [Fixed] Fix namespace packaging on Python 2. See [#11532](https://github.com/DataDog/integrations-core/pull/11532).
-* [Changed] Add tls_protocols_allowed option documentation. See [#11251](https://github.com/DataDog/integrations-core/pull/11251).
+***Changed***: 
```

Comment:
Non-blocking, does order matter for consistency? Other PRs had the following order Added -> Fixed -> Changed

Reply:
I changed the order to be in terms of priority (1. removed 2. changed 3. security 4. deprecated 5. added 6. fixed)